### PR TITLE
[master] Forward-port upgrade process changes from 1.12.x branch

### PR DESCRIPTION
**Pull request by UnderGreen**
* Update Jenkinsfile

* Change upgrade3lts function name to upgrade

* Increase timeout for golangci

----

#

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -34,8 +37,11 @@ properties([
     string(name: 'REPEAT_TESTS',
            defaultValue: '1',
            description: 'How many times to repeat each test.'),
+    string(name: 'RETRIES',
+           defaultValue: '0',
+           description: 'How many times to retry each failed test'),
     string(name: 'ROBOTEST_VERSION',
-           defaultValue: 'stable-gce',
+           defaultValue: 'uid-gid',
```
Comment (wadells) [inline Jenkinsfile:44]: I released 2.1.1 (compatible with 5.0.x) today.  It is available if you want to pick it up.

File: resources/charts/stolon/templates/alerts-create-job.yaml [hunk 2]
```diff
@@ -26,9 +27,11 @@ spec:
         - name: copy
           image: "{{ .Values.registry }}{{ .Values.image.repository }}:{{ .Values.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          # TODO(Sergei): delete `exit 0` after properly implementing Prometheus alert
           command: ["/usr/bin/dumb-init", "--", "/bin/bash", "-cex"]
           args:
             - |
+              /usr/bin/gravity resource rm alert stolon-replication-lag && exit 0 && \
```
Comment (pablodcar) [inline resources/charts/stolon/templates/alerts-create-job.yaml:34]: The intention is not to create the alerts if the alert `stolon-replication-lag` is removed

Comment (UnderGreen) [reply resources/charts/stolon/templates/alerts-create-job.yaml:34]: I have no chance to rewrite stolon alert into prometheus. This one is just a temporary thing to fix the upgrade.

Comment (pablodcar) [reply resources/charts/stolon/templates/alerts-create-job.yaml:34]: ok, makes sense

**Pull request merged**